src/aura/autocomplete/autocomplete.cmp

```diff
@@ -77,6 +77,7 @@
                                   class="{!'slds-input ' + (and(v.showPills, !empty(v.value), !empty(v.displayValue)) ? 'slds-hide' : '')}"
                                   placeholder="{!v.placeholder}"
                                   value="{!v.displayValue}"
+                                  updateOn="keyup"
```

Comment:
if they paste with the mouse, then they must hit enter to get it to requery, which seems reasonable enough to me.